src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java

```diff
@@ -319,6 +335,26 @@ public void verifyComponentRemoved(Components component) {
     PageObjectLogging.log("verifyGalleryRemoved", "Click on 'remove button' on gallery", true);
   }
 
+  public void removePortableInfobox() {
+    verifyComponent(portableInfoboxTransclusion);
+    driver.executeScript("$('.cke_wysiwyg_frame').contents().find('.placeholder-double-brackets .portable-infobox').mouseenter()");
+
+    removeInfoboxButton.click();
+    removeConfirmationButton.click();
+  }
+
+  public boolean checkPortableInfoboxIsNotPresent() {
+    driver.switchTo().frame(iframe);
+    try {
+      wait.forElementNotPresent(portableInfoboxBy);
+      return true;
+    } catch (TimeoutException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)